lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -98,6 +98,12 @@ def self.tags_for(item)
         tags
       end
 
+      def target_region
+        target = resource ? resource[:region] || region : region
```

Comment:
Where does `region` come from? Should this method have a parameter list `(region = default_region)`?